src/main/java/com/google/googleidentity/oauth2/client/ClientDetails.java

```diff
@@ -0,0 +1,1441 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Remove this file. It was added to the branch before the .gitignore entry (which won't automatically remove it once it's already there, it just prevents it from being added in the future).